<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddqaForm</class>
 <widget class="QWidget" name="AddqaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soru &amp; Cevap Listesi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Değişiklikleri Uygula</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revertButton">
       <property name="text">
        <string>Değişiklikleri Geri Al</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addqaButton">
       <property name="text">
        <string>Yeni Soru&amp;&amp;Cevap Ekle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showAnswersPushButton">
       <property name="text">
        <string>Cevapları Göster</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>Seçili Satırları Sil</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
